openstudiocore/ruby/CMakeLists.txt

```diff
@@ -117,6 +117,7 @@ endif()
 
 install(TARGETS openstudio_rb DESTINATION Ruby COMPONENT "RubyAPI")
 install(FILES openstudio.rb DESTINATION Ruby COMPONENT "RubyAPI")
+install(DIRECTORY openstudio DESTINATION Ruby COMPONENT "RubyAPI")
```

Comment:
@nllong you can add your try/catch block for the measure tester in the following file.  It will be embedded into the CLI and distributed in the installer:

https://github.com/NREL/OpenStudio/blob/develop/openstudiocore/ruby/openstudio/measure/ShowRunnerOutput.rb

Reply:
Not sure if this is really the best approach though, are there use cases where you want to include ShowRunnerOutput.rb but not activate that gem or vice versa?